PopulationDensity.ipynb

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
<p>Do we want to store the data in lat-lon format? Wouldn't it be easier to just convert the latitude-longtitudes of our nodes to integer valued grid coordinates when looking for the pop density instead of trying to find the closest lat-lon point?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laitalaj/railanalysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>